README.md

```diff
@@ -19,3 +19,18 @@ Installation
 ------------
 
 `qutip-cupy` is not yet officially released.
+
+If you want to try out the package you will need to have a CUDA enabled GPU, `QuTiP >5.0.0` and `CuPy`.
+We recommend using a conda environment `Python >= 3.7`.
+To install `CuPy` we recommend the following steps:
+
+- conda install cudatoolkit
+- conda install -c conda-forge cupy
+
+To install `QuTiP >5.0.0` while it is not yet released we recommend:
+
+- python -mpip install git+https://github.com/qutip/qutip.git@dev.major
```

Comment:
Just a nitpick: I don't like the Linux style (or whatever you call) argument invocation 😅
```suggestion
- python -m pip install git+https://github.com/qutip/qutip.git@dev.major
```

Reply:
We have that type of invocations all over the `yml`s

Reply:
Ah, ok, let's ignore this and make the style consistent then.

Reply:
I find the `-mpip` weird too, but as @MrRobot2211 says, it does appear to be fairy ubiquitous these days. ;)